src/components/Table/helpers/constants.tsx

```diff
@@ -0,0 +1,31 @@
+/** Constants */
+import React from 'react';
+export const ID = 'id';
+export type ID = typeof ID;
+
+export const PARENT_ID = 'parent_id';
+export type PARENT_ID = typeof PARENT_ID;
+
+export const ROOT_PARENT_ID = null;
+export type ROOT_PARENT_ID = typeof ROOT_PARENT_ID;
+
+export const LINKER_ITEM_CSS_CLASS = 'dd-linker-item';
+export type LINKER_ITEM_CSS_CLASS = typeof LINKER_ITEM_CSS_CLASS;
+
+export const CLICKABLE_CSS_CLASS = 'dd-linker-item dd-clickable';
+export type CLICKABLE_CSS_CLASS = typeof CLICKABLE_CSS_CLASS;
+
+export const CARET_CSS_CLASS = 'dd-caret';
+export type CARET_CSS_CLASS = typeof CARET_CSS_CLASS;
+
+export const CARET = <span className={CARET_CSS_CLASS}>&nbsp;&#9660;</span>;
+export type CARET = typeof CARET;
```

Comment:
Nice, first off, I had to look it up b/c [I never realized there were so many different spellings](https://grammarist.com/usage/carrot-carat-karat-caret/) of `caret`!

Reply:
Nit-pick: could we call this like `CARET_SPAN` since it's not just a string?

Reply:
cool

Reply:
sure